high_availability_guide/_topics/installation.md

```diff
@@ -34,21 +28,19 @@ the non-database {{ site.data.product.title_short }} appliances.
 
 5.  In the appliance console, configure the following:
 
-    1.  Configure the hostname by selecting **Set Hostname**.
+    1.  Select **Configure Application**.
 
-    2.  Select **Configure Application**.
-
-    3.  Select **Create key** to create the encryption key. You can
+    2.  Select **Create key** to create the encryption key. You can
         create a new key, or use an existing key on your system by
         selecting **Fetch key from remote machine** and following the
         prompts.
 
-    4.  Select **Create Internal Database**.
+    3.  Select **Create Internal Database**.
 
-    5.  Select the database disk. {{ site.data.product.title_short }} then activates
+    4.  Select the database disk. {{ site.data.product.title_short }} then activates
         the configuration.
 
-    6.  For **Should this appliance run as a standalone database
+    5.  For **Should this appliance run as a standalone database
         server?**, select `y`. Selecting this option configures this
```

Comment:
Select `y` should be:

Select **y**.

Reply:
Are there more of these?  It feels like they should be in a different PR since it's unrelated to the changes made here.